### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by sils1297** [duckduckgo/zeroclickinfo-goodies#3022]
As part of https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2289 I have tried a bit more code analysis on your project, among that our invalid link detection. (I won't make a PR because there's a high false positive rate - so far but this one's true:)

```
share/goodie/mac_address/oui_update.sh
|   5| wget•-O•-•http://www.ieee.org/netstorage/standards/oui.txt•|•sed•'
|    | [NORMAL] InvalidLinkBear:
|    | Broken link - unable to connect to http://www.ieee.org/netstorage/standards/oui.txt (HTTP Error: 404)
|    | The following actions are applicable to this result:
|    |  0: Apply no further actions.
|    |  1: Open the affected file(s) in an editor.
|    | Please enter the number of the action you want to execute. 0
```

You might want to check that.

**Comment by tagawa** [comment]
Thanks  for reporting this @sils1297. It looks like http://standards-oui.ieee.org/oui/oui.txt might be the correct replacement URL.

**Comment by edgesince84** [comment]
can this be closed by #3370?

**Comment by tagawa** [comment]
I reckon so @edgesince84 - thanks.
